lib/jsdom/events/EventTarget.js

```diff
@@ -0,0 +1,189 @@
+﻿"use strict";
+
+var EventException = require("./EventException");
+
+var propEvents = Symbol();
+
+module.exports = EventTarget;
+
+function EventTarget() {
+  if (!this.__jsdom) {
```

Comment:
Wait you need this even for non-`Window`s? I would just put the symbols directly on `this`...